.gitlab-ci.yml

```
@@ -682,6 +682,35 @@ zombienet-tests-parachains-disputes:
   tags:
     - zombienet-polkadot-integration-test
 
+zombienet-test-parachains-upgrade-smoke-test:
+  stage:                           deploy
```

```suggestion
  stage:                           stage4
```
Stages were renamed and became artificial in order to use DAG in the pipeline.

---

why stage4? Shouldn't it be the same stage as other zombienet tests, which is stage3?

---

fully agree, fixed suggestion